### PR TITLE
[TIMOB-25353] Android: Use ti.playservices module

### PR DESCRIPTION
**Pull request by garymathews**
- Use `ti.playservices` module
- 80% reduction in module size

###### [ti.map-4.0.0](https://github.com/appcelerator-modules/ti.map/releases/download/android-4.0.0/ti.map-android-4.0.0.zip)

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-25353)

### CODE REVIEW

File: android/timodule.xml [hunk 1]
```diff
@@ -1,27 +1,14 @@
...
+         <uses-permission android:name="${tiapp.properties['id']}.permission.MAPS_RECEIVE"/>
+         <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
+         <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
+      </manifest>
 	</android>
-	<mobileweb>
-	</mobileweb>
+   <modules>
+      <module platform="android">ti.playservices</module>
+   </modules>>
```
Comment (hansemannn) [inline android/timodule.xml]: Typo? Tested already?

Comment (garymathews) [reply android/timodule.xml]: Good catch! It still worked 👍

**Comment by sgtcoolguy** [comment]
@garymathews I think you'll need to update the sdk used to build this to be `7.0.0.v20171108185342` in the Jenkinsfile and then it should work...

**Comment by hansemannn** [comment]
@sgtcoolguy The build fails due to [this change](https://github.com/appcelerator/titanium_mobile/blob/master/android/cli/commands/_buildModule.js#L327) which looks to be added recently:
```js
[ERROR] :  Failed to run command "build"
[ERROR] :  TypeError: Path must be a string. Received 0
    at assertPath (path.js:7:11)
    at Object.join (path.js:1218:7)
    at AndroidModuleBuilder.obtainModuleDependency (/Users/build/Library/Application Support/Titanium/mobilesdk/osx/7.0.0.v20171110073543/android/cli/commands/_buildModule.js:327:19)
    at Array.forEach (native)
    at AndroidModuleBuilder.initialize (/Users/build/Library/Application Support/Titanium/mobilesdk/osx/7.0.0.v20171110073543/android/cli/commands/_buildModule.js:338:53)
    at /Users/build/Library/Application Support/Titanium/mobilesdk/osx/7.0.0.v20171110073543/node_modules/async/dist/async.js:3816:26
    at replenish (/Users/build/Library/Application Support/Titanium/mobilesdk/osx/7.0.0.v20171110073543/node_modules/async/dist/async.js:982:17)
    at iterateeCallback (/Users/build/Library/Application Support/Titanium/mobilesdk/osx/7.0.0.v20171110073543/node_modules/async/dist/async.js:967:17)
    at /Users/build/Library/Application Support/Titanium/mobilesdk/osx/7.0.0.v20171110073543/node_modules/async/dist/async.js:944:16
    at /Users/build/Library/Application Support/Titanium/mobilesdk/osx/7.0.0.v20171110073543/node_modules/async/dist/async.js:3821:13
    at apply (/Users/build/Library/Application Support/Titanium/mobilesdk/osx/7.0.0.v20171110073543/node_modules/async/dist/async.js:21:25)
    at /Users/build/Library/Application Support/Titanium/mobilesdk/osx/7.0.0.v20171110073543/node_modules/async/dist/async.js:56:12
    at AndroidModuleBuilder.doAnalytics (/Users/build/Library/Application Support/Titanium/mobilesdk/osx/7.0.0.v20171110073543/android/cli/commands/_buildModule.js:259:2)
    at /Users/build/Library/Application Support/Titanium/mobilesdk/osx/7.0.0.v20171110073543/node_modules/async/dist/async.js:3816:26
    at replenish (/Users/build/Library/Application Support/Titanium/mobilesdk/osx/7.0.0.v20171110073543/node_modules/async/dist/async.js:982:17)
    at iterateeCallback (/Users/build/Library/Application Support/Titanium/mobilesdk/osx/7.0.0.v20171110073543/node_modules/async/dist/async.js:967:17)
```

**Pull request merged**